src/rust/en.zeroscans/src/helper.rs

```diff
@@ -0,0 +1,25 @@
+use aidoku::std::{current_date, String};
+
+pub fn get_date(time_ago: String) -> f64 {
+	let cleaned_time_ago = String::from(time_ago.replace("ago", "").trim());
+
+	let number = cleaned_time_ago
+		.split_whitespace()
+		.next()
+		.unwrap_or("")
+		.parse::<f64>()
+		.unwrap_or(0.0);
+
+	match cleaned_time_ago
+		.to_uppercase()
+		.split_whitespace()
+		.last()
+		.unwrap_or("")
+	{
+		"YEAR" | "YEARS" => current_date() - (number * 31556926.0),
+		"MONTH" | "MONTHS" => current_date() - (number * 2629743.0),
+		"WEEK" | "WEEKS" => current_date() - (number * 604800.0),
+		"DAY" | "DAYS" => current_date() - (number * 86400.0),
```

Comment:
probably better to do the seconds per period multiplied out rather than constant numbers so it's clearer. e.g. `60 * 60 * 24 (=86400)` for 60s * 60m * 24hr = 86400s in a day, then add `* 7` for a week, etc.